CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
   * Kafka Connect, Mirror Maker, Mirror Maker 2 and Strimzi Bridge can be configured to use OpenTelemetry
   * Using Jaeger exporter by default for backward compatibility
 * Updated JMX Exporter dependency to 0.17.2
+* Support multiple operations per ACLRule
```

Comment:
Maybe you can also add to the subchapter below a note about the deprecation of the single operation field.